app/main.py

```diff
@@ -1,6 +1,20 @@
+import requests
+import os
+
+KEY = os.environ["API_KEY"]
+URL = f"https://api.weatherapi.com/v1/current.json?key={KEY}&q=Paris"
```

Comment:
Create a constant to store a city
In this case, you can easily change it if you want